### PR TITLE
Allow to configure the base domain used for the snc cluster

### PR DESCRIPTION
**Pull request by cfergeau**
The first 3 commits are unrelated, but they are small enough that they did not seem worth a PR of their own ;)

### CODE REVIEW

File: crc_libvirt.template [hunk 1]
```diff
@@ -93,12 +93,12 @@ cluster_create()
   </forward>
   <bridge name='tt0' stp='on' delay='0'/>
   <mac address='52:54:00:76:98:81'/>
-  <domain name='ReplaceMeWithCorrectVmName.tt.testing' localOnly='yes'/>
+  <domain name='ReplaceMeWithCorrectVmName.ReplaceMeWithCorrectBaseDomain/>
```
Comment (praveenkumar) [inline crc_libvirt.template]: why you removed `localOnly='yes'` this is important to have local dns communication.

Comment (cfergeau) [reply crc_libvirt.template]: This got removed by mistake, I'll readd

File: crc_libvirt.template [hunk 2]
```diff
@@ -93,12 +93,12 @@ cluster_create()
   </forward>
   <bridge name='tt0' stp='on' delay='0'/>
   <mac address='52:54:00:76:98:81'/>
-  <domain name='ReplaceMeWithCorrectVmName.tt.testing' localOnly='yes'/>
+  <domain name='ReplaceMeWithCorrectVmName.ReplaceMeWithCorrectBaseDomain localOnly='yes'/>
```
Comment (cfergeau) [inline crc_libvirt.template]: Hrm, now I'm missing a closing ' :)

File: install-config.yaml [hunk 3]
```diff
@@ -1,7 +1,7 @@
 # This file was generated using openshift-install create install-config
 # and then user specific information was removed as snc.sh will readd it
-apiVersion: v1beta4
-baseDomain: tt.testing
+apiVersion: v1
```
Comment (praveenkumar) [inline install-config.yaml]: 👍

**Pull request merged**